types/didChangeDependencyPaths.ts

```diff
@@ -0,0 +1,25 @@
+export const DID_CHANGE_DEPENDENCY_PATHS_NOTIFICATION_METHOD = 'aws/didChangeDependencyPaths'
+
+/**
+ * Parameters for notifying AWS language server about dependency paths
+ */
+export interface DidChangeDependencyPathsParams {
+    /** Name of the module being processed */
+    moduleName: string
+    /** Programming language runtime (e.g., 'javascript', 'python', 'java') */
+    runtimeLanguage: string
```

Comment:
I'm still not sure that "language" should be specified here. When should the client send "javascript" instead of "typescript"? In that case, the runtime (nodejs) is more relevant, because that also gives a hint about the package manager.

Reply:
> For a node project with both js and ts files, the client can send one notification for each language, imo this is not redundant but rather being explicit in the communication. I am open to discuss

I thought we discussed that the state for dependency tracking was defined exactly once for a workspace? Thus multiple requests overwrite the previous request.

If that's not the case, then the docstring here should mention "the client must send one notification for each language".